### PR TITLE
Adds fully customizable main menu

### PR DESCRIPTION
**Pull request by edward-springload**
- New models for menu items and sub menu items
 - Old secondary menu models removed
 - Model Admin for editing and adding menu items and links
 - tests for menu
 - tests for blog page next and previous links

### CODE REVIEW

File: ietf/utils/models.py [hunk 1]
```diff
@@ -66,153 +68,144 @@ class PromoteMixin(models.Model):
...
+        PageChooserPanel("page"),
+        InlinePanel(
+            "sub_menu_items",
+            label="Sub Menu Items",
+        ),
     ]
 
+    @property
+    def is_dropdown(self):
+        return bool(self.sub_menu_items.count())
```
Comment (haydnnewport) [inline ietf/utils/models.py:148]: You can use `.exists()` here instead -- it returns a bool, and is generally more efficient (though would be trivially different here)

**Review by haydnnewport — APPROVED** [review]
Looks good to me :+1:

**Comment by edward-springload** [comment]
Issue #96 

Testing for issue #38

File: requirements.txt [hunk 2]
```diff
@@ -14,6 +14,7 @@ django_analytical>=2.5
 tqdm>=3.7.0
 wagtail-markdown
 Pygments
+wagtail-orderable
```
Comment (kesara) [inline requirements.txt:17]: **wagtail-orderable** seems to be not supported.
Reference: https://github.com/elton2048/wagtail-orderable/issues/30
Is there an alternative for this dependency?

Comment (edward-springload) [reply requirements.txt:17]: Not as far as I know.

This is one of those projects that seems to just be "complete" while being small enough to bugfix if there are future issues.

Comment (kesara) [reply requirements.txt:17]: In that case, let's keep this dependency.

**Review by kesara — APPROVED** [review]

**Pull request merged**